perf/benchmark/runner/generate_policies/generate.go

```diff
@@ -1,55 +1,53 @@
 package main
 
 import (
-	"fmt"
-	"bytes"
-	"math/rand"
+	// "fmt"
+	// "bytes"
+	// "math/rand"
+	authzpb "istio.io/api/security/v1beta1"
 )
 
 type generator interface {
-    generate(kind string, num int) (string, error)
+    generate(kind string, num int) (*authzpb.Rule, error)
 }
 
 type operationGenerator struct {
 }
 
-func (operationGenerator) generate(kind string, num int) (string, error) {
-	return "", fmt.Errorf("unimplimented")
+func (operationGenerator) generate(kind string, num int) (*authzpb.Rule, error) {
+	// TODO implement
+	condition := &authzpb.Rule{}
+    return condition, nil
 }
 
 type conditionGenerator struct {
 }
 
-func (conditionGenerator) generate(kind string, num int) (string, error) {
-	rule := bytes.Buffer{}
-	rule.WriteString("  - key: \n")
-
-    for i := 0; i < num; i++ {
-		rule.WriteString("    value")
+func (conditionGenerator) generate(kind string, num int) (*authzpb.Rule, error) {
+	rule := &authzpb.Rule{}
+	listCondition := make([]*authzpb.Condition, 0)
+
+	for i := 0; i < num; i++ {
+		condition := &authzpb.Condition{}
+		condition.Key = "request.headers[x-token]"
+		var values []string
+		values = append(values, "admin")
+		values = append(values, "guest")
+		condition.NotValues = values
```

Comment:
hard-coding to set the `NotValues` field is fine for now but we need to change to generate dynamically based on the input argument later. Same for conditions and the "request.headers".